eoscostarica/eosio-spend-explorer
# feat(webapp): improve UI from main page

###  Improve UI from main page 

### What does this PR do?

- Resolve #53 

### Steps to test

1. run the project
2. open on your browser localhost:3000
3. click on icome in the sidebar and see the changes

#### CheckList

- [] Follow proper Markdown format
- [] The content is adequate
- [] The content is available in both english and spanish
- [] I Ran a spell check
